### ISSUE TITLE
compile() deprecated since gradle 8.0 in /src/android/build/localnotification.gradle

### ISSUE THREAD

**Issue by rgarciadelongoria** [katzer/cordova-plugin-local-notifications#1977]
**WARNING: IF YOU IGNORE THIS TEMPLATE, WE'LL IGNORE YOUR ISSUE. YOU MUST FILL THIS IN!**

compile() deprecated since gradle 8.0 in /src/android/build/localnotification.gradle

You fixed this same error in another of your repositories https://github.com/katzer/cordova-plugin-badge/commit/8e870f7f358d9435180f92dba4d2fc3b7f52bb4f

I have requested a pull request for this https://github.com/katzer/cordova-plugin-local-notifications/pull/1978

## Your Environment

* Plugin version: master
* Platform: all
* OS version: -
* Device manufacturer / model: -
* Cordova version (```cordova -v```): -
* Cordova platform version (```cordova platform ls```): -
* Plugin config
* Ionic Version (if using Ionic)

## Expected Behavior

no error

## Actual Behavior

<img width="1114" alt="Captura de Pantalla 2022-08-02 a las 11 16 02" src="https://user-images.githubusercontent.com/84543810/182339495-fafeb500-274d-4113-b6b5-4aee9ee6661a.png">

## Steps to Reproduce

run with capacitor

_Reproduce this issue; include code to reproduce, if relevant_

  1. ...
  2. ...
  3. ...
  4. ....

## Context

_What were you trying to do?_

## Debug logs

_Include iOS / Android logs_

  * ios XCode logs
  * Android: $ adb logcat

**Comment by jeanteke** [comment]
Hello friend, were you able to find the solution?

**Comment by rgarciadelongoria** [comment]
> 

I have requested a pull request for this https://github.com/katzer/cordova-plugin-local-notifications/pull/1978

Meanwhile I use the fork that I have created to do the PR.

**Comment by jeanteke** [comment]
I could give you the command?

**Comment by rgarciadelongoria** [comment]
> 

Sorry, what command? I don't understand.

**Comment by jeanteke** [comment]
Ro add the plugin with the fork you created

**Comment by rgarciadelongoria** [comment]
> To add the plugin with the fork you created

Ah ok.

This is the npm package https://www.npmjs.com/package/@rgarciadelongoria/firebase-remote-config

**Comment by jeanteke** [comment]
it would be like this?

cordova plugin add https://github.com/rgarciadelongoria/cordova-plugin-local-notifications.git

**Comment by rgarciadelongoria** [comment]
You can use like this.

`"cordova-plugin-local-notification": "github:rgarciadelongoria/cordova-plugin-local-notifications",`

or like this if you want to use the exact commit

`"cordova-plugin-local-notification": "github:rgarciadelongoria/cordova-plugin-local-notifications#eda42850fc310bacee7afc08a179ab46e6c3108b",`

**Comment by jeanteke** [comment]
> 

lets build the app, but when launching local notifications the app closes

**Comment by rgarciadelongoria** [comment]
> It lets build the app, but when launching local notifications the app closes

As soon as I have some time I will do a test. However, I encourage you to comment on more details, such as if it happens on Android or iOS, include an image and share the error it gives. Surely they say something about the error on stackoverflow.

Cheers

**Comment by alanmilinovic** [comment]
It is on Android in my case.

**Comment by jeanteke** [comment]
> 
You can already solve, with this other: 

`ionic cordova plugin add cordova-plugin-local-notification-12`

**Comment by rgarciadelongoria** [comment]
> It is on Android in my case.

you are right

In Android there is no .local variable and it gives an error.

In my case I made a fork and pull request of this so that it would not give an error when compiling a project I am working on, but I just saw that this plugin is not really being used and I am going to proceed to remove it from the project.

<img width="447" alt="Captura de Pantalla 2022-09-28 a las 11 41 53" src="https://user-images.githubusercontent.com/84543810/192746474-15d7769a-de08-4304-baae-78fe738bb572.png">

In ios it works fine.

> You can already solve, with this other:

> `ionic cordova plugin add cordova-plugin-local-notification-12`

Regarding the use of this fork ([cordova-plugin-local-notification-12](https://www.npmjs.com/package/cordova-plugin-local-notification-12)), it seems to me a good choice. It seems updated and it works fine.

I will close this issue.

Thank you all for participating.
